core/kazoo_stdlib/include/kz_types.hrl

```diff
@@ -275,5 +275,24 @@
 -define(DEV_LOG(F, A), io:format(user, "~s:~p  " ++ F ++ "\n", [?MODULE, ?LINE | A])).
 -define(DEV_LOG(F), ?DEV_LOG(F, [])).
 
+%% From https://github.com/tomas-abrahamsson/gpb/issues/134#issuecomment-386892877
+%% Usage:
+%% try
+%%     ...
+%% catch
+%%     error:badarg ->
+%%         whatever;
+%%     ?STACKTRACE(E, R, Stack)
+%%         {error, {E,R, Stack}}
+%% end
+%% kz.mk defines the macro if OTP version is >= 21
+-ifdef(OTP_RELEASE).
```

Comment:
Why would this be needed? 4.3 is targeted to OTP 19.3, no?

Reply:
It is not necessary needed in kazoo, I needed this for my code to support both 19 and 21.